packages/merge-on-green/src/merge-on-green.ts

```diff
@@ -464,11 +477,21 @@ handler.scanForMissingPullRequests = async function scanForMissingPullRequests(
       const ownerAndRepoArray = issue.repository_url.split('/');
       const owner = ownerAndRepoArray[ownerAndRepoArray.length - 2];
       const repo = ownerAndRepoArray[ownerAndRepoArray.length - 1];
+      let baseBranch;
+      try {
+        baseBranch = (
+          await github.pulls.get({owner, repo, pull_number: issue.number})
+        ).data.base.ref;
+      } catch (err) {
+        baseBranch = 'master';
```

Comment:
I would be tempted to drop this `try`/`catch` unless there's a known configuration where we expect the error to happen.